src/clj/rems/api/services/util.clj

```diff
@@ -0,0 +1,46 @@
+(ns rems.api.services.util
+  (:require [clojure.test :refer [deftest is testing]]
+            [rems.db.roles :as roles]
+            [rems.db.users :as users]))
+
+(defn forbidden-organization? [user-id organization]
+  (let [user (users/get-user user-id)]
+    (when (and (contains? (roles/get-roles user-id) :organization-owner)
+               (or (not= organization (:organization user))
+                   ;; XXX: Special case to forbid an organization owner with
+                   ;;   no organization defined from creating items with
+                   ;;   no organization defined. Consider disallowing undefined
+                   ;;   organizations to be able to remove this.
+                   (and (nil? organization) (nil? (:organization user)))))
+      {:success false
+       :errors [{:type :t.administration.errors/forbidden-organization}]})))
+
+(deftest test-forbidden-organization?
+  (with-redefs [roles/get-roles {"owner" #{:owner}
+                                 "organization-owner" #{:organization-owner}
+                                 "organization-owner-with-no-organization" #{:organization-owner}}
+                users/get-user {"owner" {:eppn "owner"
+                                         :organization "own organization"}
+                                "organization-owner" {:eppn "organization-owner"
+                                                      :organization "own organization"}
+                                "organization-owner-with-no-organization" {:eppn "organization-owner-with-no-organization"}}]
+    (let [forbidden-error {:success false
+                           :errors [{:type :t.administration.errors/forbidden-organization}]}]
+      (testing "for owner, all organizations are permitted"
```

Comment:
should also test multiple roles, e.g. owner and organization-owner

Reply:
Done.